### DISCUSSION TITLE
Operation System

### DISCUSSION THREAD

**Discussion by ogustavo-pereira** [cerebroapp/cerebro#674] [Polls]
We want to measure our audience and better understand which are the main platforms that use our app.

**Comment by twoprops** [comment]
I can't really say that I'm "using" Cerebro anywhere, as I cannot get it to run: Linux Fedora 39, built from source without errors, executing the appimage file just hangs with no error messages. CTRL-space does nothing (well, in Terminal it displays "^@").

**Reply by ogustavo-pereira** [reply]
Thank you for the feedback, this dissatisfaction is very important for us to know where we need to pay attention. Would you happen to intend to use it?

**Comment by twoprops** [comment]
I have managed Linux servers for decades, but used macOS for my desktop. I recently decided to migrate my desktop to a Framework machine running Fedora 39. There are only two macOS tools I really miss, BBEdit and LaunchBar. I ran across Cerebro when searching for a LaunchBar alternative, and it appears that it will do what I want.

I cloned the github repo, installed dependencies, built from source without errors, and installed the executable. When I run it, it just hangs. The hotkey does nothing and no icon shows up on the menu bar. No error messages are displayed in the Terminal.

I had to move on to getting my system in a usable state, so I didn't investigate much further at the time, but I would really like to see if I can get Cerebro running in my environment. Any ideas where I might look to help diagnose the problem?

EDIT: I just found this issue https://github.com/cerebroapp/cerebro/issues/291 from 2017. Apparently Cerebro isn't compatible with Wayland?
